churaverse-plugins-client/src/webRtcPlugin/webRtc.ts

```diff
@@ -26,18 +26,21 @@ export class WebRtc {
       },
     }
     this.room = new Room(roomOptions)
-
     void this.connect(ownPlayerId)
   }
 
   private async connect(ownPlayerId: string): Promise<void> {
     try {
       const token = await this.getAccessToken(ownPlayerId)
-      await this.room.connect(`${import.meta.env.VITE_LIVEKIT_URL ?? 'ws://localhost:8080/livekit'}`, token)
-
+      const iceServers = [{ urls: ['stun:stun.l.google.com:19302'] }]
+      await this.room.connect(`${import.meta.env.VITE_LIVEKIT_URL ?? 'ws://localhost:8080/livekit'}`, token, {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-expect-error
+        iceServers,
+      })
```

Comment:
eslintで無視させずに以下のように書いた方がわかりやすそうです。

```suggestion
            await this.room.connect(`${import.meta.env.VITE_LIVEKIT_URL ?? 'ws://localhost:8080/livekit'}`, token, {
        rtcConfig: { iceServers },
      })
```